src/evaluate.cpp

```diff
@@ -788,7 +788,7 @@ namespace {
                 bb = forward_file_bb(Them, s) & pos.pieces(ROOK, QUEEN);
 
                 if (!(pos.pieces(Them) & bb))
-                    unsafeSquares &= attackedBy[Them][ALL_PIECES];
+                    unsafeSquares &= attackedBy[Them][ALL_PIECES] | pos.pieces(Them);
```

Comment:
Maybe we need a comment since `unsafeSquares` may contain a safe (but blocked) square.

Reply:
Thanks for the suggestion, please feel free to add the comment that you think its most suitable, or if you can't edit it, write it to me, and I will add it in the patch

Reply:
I'm a bit confused.  Wouldn't blocked be
```unsafeSquares &= attackedBy[Them][ALL_PIECES] | (pos.pieces(Them) & forward_file_bb(Them, s));```
otherwise we are including adjacent files also ?

Reply:
Well, for bloking only, it would be forward_file_bb(Us, s)

Anyway, correct comment for current patch would be
// If there are no enemy pieces or attacks on passsed pawn span, assign a big bonus.

Reply:
@FauziAkram can you add the suggested comment to the PR.

Reply:
@vondele I did add it 2 days ago